### ISSUE TITLE
Abacus is printing DB username and password in the logs

### ISSUE THREAD

**Issue by AzureEnigma** [cloudfoundry-incubator/cf-abacus#141]
When you turn on debugging, abacus is printing the database and password.

```
2015-10-30T16:01:12.48-0700 [App/0]      OUT   _db_name: 'https://db_username:db_password@db_url/database',
```

**Comment by jsdelfino** [comment]
They're logged because they're configured in the DB url.

We'll need to add support for specifying the db user and password in env variables outside of that URL like follows:
```
DB_USER=...
DB_PASSWORD=...
```

**Comment by jsdelfino** [comment]
@AzureEnigma Which DEBUG log did you have on when you saw the user and password in the log? Thanks!

**Comment by AzureEnigma** [comment]
The DEBUG that i was using is DEBUG="abacus-*". In the aggregator, this will print out the database credidentials.

**Comment by jsdelfino** [comment]
Fixed by commits 1af2b2f (pull request #186) and subsequent commit c451fc7.
